composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "liip/metadata-parser",
+    "description": "Parser for metadata",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Liip AG",
+            "homepage": "http://www.liip.ch"
+        }
+    ],
+    "support": {
+        "source": "https://github.com/liip/metadata-parser",
+        "issues": "https://github.com/liip/metadata-parser/issues"
+    },
+    "require": {
+        "php": "^7.1",
+        "psr/log": "^1",
+        "jms/serializer": "^2"
```

Comment:
this is only necessary when using the JMSParser. i would put that into require-dev and suggestion.